chart/flux/templates/deployment.yaml

```diff
@@ -69,6 +69,15 @@ spec:
           secretName: {{ .Values.gpgKeys.secretName }}
           defaultMode: 0400
       {{- end }}
+      {{- if .Values.extraVolumes }}
+{{ toYaml .Values.extraVolumes | indent 8 }}
+      {{- end }}
+{{- if .Values.initContainers }}
+      initContainers:
+{{- range $key, $value := .Values.initContainers }}
+        - name: {{ $key }}
+{{ toYaml $value | indent 10 }}
+{{- end }}
```

Comment:
Is there a missing `{{- end }}` here (for the `if .Values.initContainers`)?

Reply:
Added this but still not happy.